### ISSUE TITLE
Use in java

### ISSUE THREAD

**Issue by tellfa** [iammert/AnimatedTabLayout#2]
Thanks for this great lib.
Can i use this into java?

**Comment by frknkntr** [comment]
Yes, you can use this library in java project. But you must add kotlin to your project.

**Comment by akhileshsharma** [comment]
Issue in Java implementation. on click of tabs does not work.  In debug mode code does not enter to code block where it add onClickListner for tabs in AnimatedTabLayout.kt

tabs.forEach { tab ->
            containerLinearLayout.addView(tab)
            tab.setOnClickListener {
                val selectedIndex = tabs.indexOf(tab)
                onPageChangeListener.onPageSelected(selectedIndex)
                viewPager.currentItem = selectedIndex
            }
        }
